### PR TITLE
typo(cmd/mirror): add warning if key already exists

### PR DESCRIPTION
**Pull request by 9547**
### What is changed and how it works?
Add `Warning` if the private key exists

### Check List 

Tests 

 - Unit test
 - Integration test
 - Manual test (add detailed scripts or steps below)
 - No code

Code changes

 - Has exported function/method change
 - Has exported variable/fields change
 - Has interface methods change
 - Has persistent data change

Side effects

 - Possible performance regression
 - Increased code complexity
 - Breaking backward compatibility

Related changes

 - Need to cherry-pick to the release branch
 - Need to update the documentation

Release notes:

```release-note
NONE
```

### CODE REVIEW

File: cmd/mirror.go [hunk 1]
```diff
@@ -548,7 +548,7 @@ func newMirrorGenkeyCmd() *cobra.Command {
 				fmt.Printf("KeyID: %s\nKeyContent: \n%s\n", id, string(content))
 			} else {
 				if utils.IsExist(privPath) {
-					fmt.Println("Key already exists, skipped")
+					fmt.Println("Warning: Key already exists, skipped")
```
Comment (AstroProfundis) [inline cmd/mirror.go:551]: maybe use a `log.Warn()`?

**Comment by 9547** [comment]
@AstroProfundis PTAL

**Review by AstroProfundis — APPROVED** [review]

**Pull request merged**